pkg/rancher-desktop/main/commandServer/httpCommandServer.ts

```diff
@@ -117,19 +115,6 @@ export class HttpCommandServer {
 
   async init() {
     const localHost = '127.0.0.1';
-
-    // The peerPort and upstreamServerAddress port will need to match
-    // this is crucial if we ever pick dynamic ports for upstreamServerAddress
-    if (process.platform === 'win32') {
```

Comment:
Hmm, just realized that running `rdctl` from WSL (non-RD distro) doesn't work; should we file that as a bug? Old network would have worked (because of this line).